### PR TITLE
Fix/after release fixes

### LINKED ISSUE #2593: After release issues, suggestions
### Header fixes in the docs
- [x] remove icons (overview, github, contribution)
- [x] change version color to text
- [x] tapping a logo should refresh the page (not take you to the landing page)
- [x] tapping Overview  should bring you to the landing page

### Sidebar badges
Will be nice to describe when to use badges, and when chips.
- [x] use badge instead of chip for new and updated
- [x] new should have danger bg
- [x] updated should have warning bg

### Landing
- [x] Header, fix English dropdown style, so it fits other menu items
- [x] Header, tapping a logo should refresh the page (not take you to Docs)
- [x] Header, remove Overview  and add Docs  to bring users to documentation
- [x] change 52 components to 60 components
- [x] check if secondary buttons (like View in GitHub have proper bg colour that we have introduced lately for updated buttons. now it seems too dark
  image

### Buttons
Large buttons should have the same border-radius as medium  which is 4px by default
- [x] check default medium buttons to have 20x20 icons, not 24x24
- [x] check default medium buttons have 4px padding between text and icons, not 2
- [x] check same things for other button sizes

### Buttons in the docs
Update Add icons as following:
- [x] reduce margin between buttons to 12px
- [x] danger bg for the first button, update label to Delete
- [x] update label to Continue , change icon to arrow-forward and apply #fff with 50% opacity (if it’s not possible, simply apply warning
- [x] change the icon to plus , update bg to warning and icon colour to custom 812E9E
- [x] update icon in the last example to star and unfold_more

<details>
<summary>Add icons block</summary>

image
</details>

Update Text color as following:
- [x] reduce margin between buttons to 12px
- [x] change label in the first button to Default and have it as default white
- [x] add a button with 80% white as a label color if possible, the label should say Default with opacity
- [x] an example with warning text color
- [x] the last example is a custom purple 812E9E text over warning bg

<details>
<summary>Text colors block</summary>

image
</details>

### Split in the docs
- [x] use default scroll style, not blue
- [x] update custom grabber: bg to backgroundElement and swap_horiz icon to secondary

<details>
<summary>Text colors block</summary>

image
</details>

### TreeView
In Filters:
- [x] add 16px margin between search input and checkbox
- [x] center them vertically
- [x] change the placeholder to Filter…

<details>
<summary>Tree view examples</summary>

image
image
</details>

In Selectable:
- [x] add 10px bottom margin for the first chip element
- [x] remove hover for chip

<details>
<summary>Current implementation</summary>

image

</details>

### Virtual scroll
- [x] dark theme style needs a fix

image

### Code examples
- [x] apply Source Code Pro as the only font-family for code so we don’t run into multiple fonts like in the attachment

image

### Badges
- [x] badges by default should have 2px border radius, same as small buttons
- [x] Add badge (updated) to [components-config](https://vuestic.dev/en/services/components-config)

image

### PR DESCRIPTION
**Pull request by rustem-nasyrov**
Close #2593

## Description
A lot of changes: docs, examples, fixes, added to the VaChip readonly state.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Comment by xx13** [comment]
The header is docs looks nice, but the current language label is missing
image

There is still an issue with the landing page: icons should be removed, the current language label added
image

In docs, if we can add `new` badge on the right of `Colors`, that will be great!

In navigation: we shouldn't display the landing page in dark theme, even if it was selected in docs.
image

There is still an issue with the landing page navbar in Safari 14.1.2, but it Docs it works fine tho (it might be because we use different elements as navbar items).
image

The other thing I notice, is that `Open Source` section on the landing page should have `#fff` bg colour, now it is grey for some reason.

Badges in the sidebar, and Button, Tree View, Split demo updates look great to me 👍👍

**Comment by rustem-nasyrov** [comment]
There is still an issue with transparent or white labels on the buttons on the landing page (header, footer). Reproduces only in safari, for details ask @xx13.

image
image

File: packages/ui/src/components/va-badge/_variables.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@
   --va-badge-text-wrapper-transition: 0.2s cubic-bezier(0.4, 0, 0.6, 1);
   --va-badge-text-wrapper-display: inline-flex;
   --va-badge-text-wrapper-border: solid 0.125rem;
-  --va-badge-text-wrapper-border-radius: 0.6rem;
+  --va-badge-text-wrapper-border-radius: 0.125rem;
```
Comment (m0ksem) [inline packages/ui/src/components/va-badge/_variables.scss:20]: Isn't it a braking-change? Might be spooky to see badge suddenly changed one day. I would suggest changing it in 1.6.0 or we don't care?

**Review by m0ksem — APPROVED** [review]

File: packages/ui/src/components/va-badge/_variables.scss (hunk 1 above)
Comment (rustem-nasyrov) [reply packages/ui/src/components/va-badge/_variables.scss:20]: @m0ksem I think it's not that breaking change if user will mention that, he can use his own border-radius value.

**Pull request merged**